Extension/Context.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Netdudes\DataSourceryBundle\Extension;
+
+class Context
```

Comment:
The only benefit is that maybe in future it'll be possible to add more fields without breaking BC.
And passing "context" object instead of "entity class" to every function call seems to have more sense ("why you're passing an entity class to now() function?").